tests/Doctrine/Tests/TestUtil.php

```diff
@@ -186,6 +188,24 @@ private static function mapConnectionParameters(array $configuration, string $pr
             $parameters[$parameter] = $configuration[$prefix . $parameter];
         }
 
+        $defaultTableOptions = [];
```

Comment:
I would prefer the test like DDC2182Test and avoid changing the core features of tests.